synapse/config/server.py

```diff
@@ -1349,9 +1349,15 @@ def parse_listener_def(listener: Any) -> ListenerConfig:
     return ListenerConfig(port, bind_addresses, listener_type, tls, http_config)
 
 
+NO_MORE_NONE_HTTP_WEB_CLIENT_LOCATION_WARNING = """
+Synapse no longer supports serving a web client. To remove this warning, ensure
```

Comment:
I think the "ensure" might be unneeded here?

Reply:
Yeahhhh I must have reworded this sentence or something. Thanks!

```suggestion
Synapse no longer supports serving a web client. To remove this warning,
```